financialaid/api.py

```diff
@@ -11,7 +12,7 @@ def determine_tier_program(program, income):
         program (Program): the Program to determine a TierProgram for
         income (numeric): the income of the User
     Returns:
-        TierProgram: the TierProgram for the Program given the User's income
+        (TierProgram): the TierProgram for the Program given the User's income
```

Comment:
Parentheses are not used in the return value. See here: https://pypi.python.org/pypi/sphinxcontrib-napoleon